### ISSUE TITLE
Edit The "Share Your Resume" domain

### ISSUE THREAD

**Issue by stevethepirate88** [AmruthPillai/Reactive-Resume#398]
Okay. I got everything else to work, but there is one last thing I have a question about.

How can I change the domain for the "Share your resume" link? I am currently hosting this on a `resume.XXXXX.dev` domain, and I would like to be able to link to the created resume, however the share your resume box is still showing an rxresu.me link that doesn't work because this box isn't communicating with that site.
<img width="244" alt="Screen Shot 2020-12-10 at 11 06 04 AM" src="https://user-images.githubusercontent.com/22171718/101797495-1edbcb80-3ad8-11eb-9e0e-9a2ed0d2fa52.png">

If this isn't already a feature, that would be a super cool one to offer.

**Comment by AmruthPillai** [comment]
I've hard-coded the site URL there to rxresu.me, but if you have your site running on `resume.xyz.dev` then you should be able to just visit `http://resume.xyz.dev/r/:resumeId` to share your resume with everyone.

**Comment by stevethepirate88** [comment]
Hey Amruth, thank you for sharing that! I just checked, and the `http://resume.xyz.dev/r/:resumeId` domain does not seem to work. It presents me instead with an nginx 404 error. I checked the `Reactive-Resume/public/r/` folder and it only has an `index.html` file in it. I'm assuming that index.html pulls from other sources to find the the resume based on the ID. What would be the best way for me to start tracing things back to make any necessary changes to resolve the 404?
